app/forms/fields/select_field_with_detail_answer.py

```diff
@@ -27,11 +52,11 @@ def __iter__(self):
             opt.checked = checked
```

Comment:
`opt.process(None, value)` implies the `formdata` parameter elsewhere can also be None so it might be `MultiDict | None` where its defined